src/other/HeatMap.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "../common/CanvasWidget", "simpleheat"], factory);
+    } else {
+        root.other_Heat = factory(root.d3, root.common_CanvasWidget, root.simpleheat);
+    }
+}(this, function (d3, CanvasWidget, simpleheat) {
+    function Heat() {
+        CanvasWidget.call(this);
+    }
+ 
```

Comment:
Minor:  Remove blank link I like to keep all the "constructor" info in one block.